odoo/tools/translate.py

```diff
@@ -1155,9 +1155,9 @@ def trans_load(cr, filename, lang, verbose=True, create_empty_translation=False,
                                    verbose=verbose,
                                    create_empty_translation=create_empty_translation,
                                    overwrite=overwrite)
-    except IOError:
+    except IOError as io_error:
         if verbose:
-            _logger.error("couldn't read translation file %s", filename)
+            _logger.error("couldn't read translation file %s. Here's what we got back instead: %s" % (filename, io_error)
```

Comment:
And maybe even better:
```suggestion
            _logger.error("couldn't read translation file %s for the module %s. Here's what we got back instead: %s" % (filename, module_name,  io_error)
```